river/Output.zig

```diff
@@ -107,6 +114,8 @@ pub fn init(self: *Self, root: *Root, wlr_output: *c.wlr_output) !void {
 
     self.layout = try std.mem.dupe(util.gpa, u8, "full");
 
+    self.attach_mode = AttachMode.AttachTop;
```

Comment:
can use an enum literal here
```suggestion
    self.attach_mode = .top;
```